linux_pipeline/Jenkinsfile_azure_lis_hotfix_pipeline

```diff
@@ -83,13 +83,35 @@ def GetErrataKernelLink(kernelVersion) {
 
 def CheckerrataLink(errataLink) {
     def status = sh (returnStatus: true, script: """
-        wget ${errataLink}
+        wget -q ${errataLink}
         """ )
     if ( status != 0) {
         throw new Exception("${errataLink} is unreachable.")
     }
```

Comment:
Add cleanWs() after if block to delete any files downloaded using wget.

Reply:
done